packages/errors/README.md

```
@@ -0,0 +1,252 @@
+  get module() {
+    return 'EGG_MYSQL';
+  }
+}
+
+const err = FrameworkErrorFormater.formatError(new EggMysqlError('error message', '01'));
+console.log(err.message); // error message [ https://eggjs.org/zh-cn/faq/EGG_MYSQL/01 ]
+```
+
+### Available Errors
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)